interface/main/calendar/add_edit_event.php

```diff
@@ -727,12 +738,20 @@ function setEventDate($start_date, $recurrence)
 
         // EVENTS TO FACILITIES
         $e2f = (int)$eid;
+        //Tell subscribers that a new multi appointment has been set
+        $patientAppointmentSetEvent = new AppoinmentSetEvent($_POST);
+        $patientAppointmentSetEvent->eid = $e2f;  //setting the appointment id to an object
+        $GLOBALS["kernel"]->getEventDispatcher()->dispatch(AppoinmentSetEvent::EVENT_HANDLE, $patientAppointmentSetEvent, 10);
     } else {
         /* =======================================================
      *                    INSERT NEW EVENT(S)
      * ======================================================*/
 
         $eid = InsertEventFull();
+        //Tell subscribers that a new single appointment has been set
+        $patientAppointmentSetEvent = new AppoinmentSetEvent($_POST);
+        $patientAppointmentSetEvent->eid = $eid;  //setting the appointment id to an object
+        $GLOBALS["kernel"]->getEventDispatcher()->dispatch(AppoinmentSetEvent::EVENT_HANDLE, $patientAppointmentSetEvent, 10);
```

Comment:
```
$eventDispatcher->dispatch(AppoinmentSetEvent::EVENT_HANDLE, $patientAppointmentSetEvent, 10);
```
(since you created the $eventDispatcher above, might as well use it here)

Reply:
Sure will.